dev/pyproject.py

```diff
@@ -144,10 +144,10 @@ def build(skinny: bool) -> None:
                         "pyspark/ml/log_model_allowlist.txt",
                         "server/auth/basic_auth.ini",
                         "server/auth/db/migrations/alembic.ini",
-                        "recipes/resources/*",
-                        "recipes/cards/templates/*",
+                        "recipes/resources/**/*",
```

Comment:
I need some study: what's the difference between `xx/*` and `xx/**/*` ? :)

Reply:
The difference is whether the directory can be listed